modules/contact/test/tests/mortar_cartesian_lms/tests

```diff
@@ -0,0 +1,84 @@
+[Tests]
+  design = 'constraints/ComputeFrictionalForceCartesianLMMechanicalContact.md constraints/ComputeWeightedGapCartesianLMMechanicalContact.md'
+  issues = '#21641'
+  [frictionless-weighted-gap-lm]
+    type = 'Exodiff'
+    exodiff = 'frictionless-weighted-gap-lm_out.e'
+    input = 'frictionless-weighted-gap-lm.i'
+    rel_err = 5.0e-4
+    min_parallel = 4
+    max_parallel = 4
+    min_ad_size = '50'
+    valgrind = 'heavy'
+    ad_indexing_type = 'global'
+    requirement = 'Contact module shall solve a two-dimensional frictionless problem with Cartesian Lagrange multipliers.'
+  []
+  [frictionless-mortar-3d]
+    type = 'Exodiff'
+    exodiff = 'frictionless-mortar-3d_out.e'
+    input = 'frictionless-mortar-3d.i'
+    rel_err = 5.0e-4
+    max_parallel = 4
+    min_ad_size = '100'
+    ad_indexing_type = 'global'
+    mesh_mode = 'REPLICATED'
+    requirement = 'Contact module shall solve a three-dimensional frictionless problem with Cartesian Lagrange multipliers.'
+  []
+  [friction-mortar-3d]
+    type = 'Exodiff'
+    exodiff = 'frictionless-mortar-3d-friction_out.e'
+    input = 'frictionless-mortar-3d-friction.i'
+    rel_err = 5.0e-4
+    max_parallel = 4
+    min_ad_size = '100'
+    ad_indexing_type = 'global'
+    mesh_mode = 'REPLICATED'
```

Comment:
Or here?

Reply:
Yeah, it is used. Maybe I am missing your point. But I see that this is a VCP test which is run (or tried to) with a distributed mesh.

Reply:
oh, it *is* a VCP test? Basically I want to make sure we can run all non-VCP tests with distributed mesh.